core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.ts

```diff
@@ -257,6 +264,17 @@ export class TranslationModalComponent {
     return target.localName === 'p' && !mathElementsIncluded;
   }
 
+  toggleMachineTranslation(): void {
+    this.hideMachineTranslation = !this.hideMachineTranslation;
+    if (!this.hideMachineTranslation) {
```

Comment:
I think this will be easier to read if we stick to the "positive" version of the boolean, i.e. `showMachineTranslation`. "If not hide" reads like a double negative and is more difficult to reason about.